src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+
```

Comment:
- [ ] Kindly, fix eslint errors without disabling line of code.

Reply:
👍🏾 done.